build.gradle

```diff
@@ -73,10 +81,12 @@ publishing {
 			}
 		}
 	}
-
-	// Select the repositories you want to publish to
-	// To publish to maven local, no extra repositories are necessary. Just use the task `publishToMavenLocal`.
-	repositories {
-		// See https://docs.gradle.org/current/userguide/publishing_maven.html for information on how to set up publishing.
-	}
+    
+    // See https://docs.gradle.org/current/userguide/publishing_maven.html for information on how to set up publishing.
+    repositories {
+        // Add repositories to publish to here.
+        // Notice: This block does NOT have the same function as the block in line 13.
```

Comment:
I wouldnt specifcy an exact line number as its going to move and this wont get updated.

Reply:
```suggestion
        // Notice: This block does NOT have the same function as the block in the top level.
```